client/components/common/Meta/Checkbox.vue

```diff
@@ -1,24 +1,19 @@
 <template>
-  <div class="checkbox control">
+  <div class="control">
     <span class="title">{{ meta.label }}</span>
-    <div class="control-group">
-      <label :for="meta.key" :class="{ checked: value }">
-        <input
-          :ref="meta.key"
-          v-model="value"
-          @change="$emit('update', meta.key, value)"
-          :id="meta.key"
-          :name="meta.key"
-          type="checkbox">
-      </label>
-      <p class="description">{{ meta.description }}</p>
-    </div>
+    <v-checkbox
+      v-model="value"
```

Comment:
There is no need for a v-model here nor below. A combination of `input-value` prop and a `change` handler will be enough for our requirements.